src/patternfly/components/DescriptionList/description-list__example.hbs

```diff
@@ -13,23 +15,29 @@
   {{/description-list-group}}
   {{#> description-list-group}}
     {{#> description-list-term}}
-      Namespace
+      {{#> description-list-text description-list-text--type="span"}}
+        Namespace
+      {{/description-list-text}}
     {{/description-list-term}}
     {{#> description-list-description}}
       <a href="#">mary-test</a>
     {{/description-list-description}}
   {{/description-list-group}}
   {{#> description-list-group}}
     {{#> description-list-term}}
-      Labels
+      {{#> description-list-text description-list-text--type="span"}}
+        Labels
+      {{/description-list-text}}
     {{/description-list-term}}
     {{#> description-list-description}}
       example
     {{/description-list-description}}
   {{/description-list-group}}
   {{#> description-list-group}}
     {{#> description-list-term}}
-      Pod selector
+      {{#> description-list-text description-list-text--type="span"}}
```

Comment:
This works, but something we've done in other templates is wrap all of the contents of the `description-list-text` partial with a call to a partial that doesn't exist (so it doesn't render an element) to always include the context in that template. We use `{{#> wrapper}}` as the partial name. You can see an example of that here - https://github.com/patternfly/patternfly/blob/89fa7ba92f6953b048f1d95a7cced1a46b8a896b/src/patternfly/components/Table/Tree-table/table-tree-view--basic.hbs#L27-L30

Here's what we would add to the `description-list-text` partial
```
{{#> wrapper description-list-text--type="span"}}
  [original hbs contents go here]
{{/wrapper}}
```

Then we can remove `description-list-text--type="span"` (and more importantly don't have to include it in the future) for all uses of `description-list-text`  in `description-list-term`